sdk/schemaregistry/schema-registry-avro/vitest.browser.config.ts

```diff
@@ -10,9 +10,9 @@ export default mergeConfig(
   viteConfig,
   defineConfig({
     optimizeDeps: {
-      include: ["process", "buffer"],
+      include: ["buffer"],
     },
-    plugins: [browserMap(), inject({ process: "process", Buffer: ["buffer", "Buffer"] })],
+    plugins: [browserMap(), inject({ Buffer: ["buffer", "Buffer"] })],
```

Comment:
I think we need to keep these configs, looks like EH tests are failing in https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4597779&view=logs&j=ddfb31fb-6394-5939-4454-ec82d3301bb7&t=6c1cf1bd-2761-5851-504f-6f91f3dc6e27

Reply:
Fixed